root/documentation/ld.conf

```diff
@@ -1,7 +1,7 @@
 <endpoints>
   <ld_id_get>
     description=Computes and returns LD values between the given variant and all other variants in a window centered around the given variant. The window size is set to 500 kb.    
-    endpoint=ld/:species/:id/:population_name/:d_prime/:r2/:window_size
+    endpoint=ld/:species/:id/:population_name/:d_prime/:r2/:window_size/:attribs
```

Comment:
I believe this isn't correct. :variable implies part of the URL, not simply ?options you can slap on the end